torch/ao/quantization/observer.py

```diff
@@ -770,8 +770,11 @@ def _load_from_state_dict_script(
     @torch.jit.export
     def reset_min_max_vals(self):
         """Resets the min/max values."""
-        self.min_val = torch.tensor([])
-        self.max_val = torch.tensor([])
+        # This used to be torch.ones but that does not work because
```

Comment:
just to make sure this does not have effect on actual Tensor right? it's still a tensor with 0 elements?
if so I think it should be fine

Reply:
yes, that is correct.

Reply:
sg, looks good to me